src/components/signUp/signUpResponse.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Homepage from './../homepage/homepage';
+const SignUpResponse  = ({response})=>{
+
+    console.log(`Response in other file ${response.status}`);
```

Comment:
`console.log` lines should not be committed.

Reply:
yeah got it